service/src/main/java/uk/gov/hmcts/reform/fpl/service/CMODocmosisTemplateDataGenerationService.java

```diff
@@ -138,29 +143,54 @@ private CaseManagementOrder getCaseManagementOrder(CaseData caseData) {
         return null;
     }
 
-    private List<Map<String, Object>> getRepresentatives(List<Element<Respondent>> respondents1,
+    private List<Map<String, Object>> getRepresentatives(CaseData caseData,
                                                          String applicantName,
                                                          Solicitor solicitor) {
-        List<Map<String, Object>> representatives = new ArrayList<>();
 
-        representatives.add(getApplicantDetails(applicantName, solicitor));
+        List<Map<String, Object>> representativesInfo = new ArrayList<>();
+        List<Element<Representative>> representatives = caseData.getRepresentatives();
 
-        if (!isEmpty(respondents1)) {
-            respondents1.stream()
-                .map(Element::getValue)
-                .forEach(respondent -> representatives.add(getRepresentative(respondent)));
-        }
+        representativesInfo.add(getApplicantDetails(applicantName, solicitor));
+
+        ElementUtils.unwrapElements(caseData.getRespondents1()).stream()
+            .filter(respondent -> isNotEmpty(respondent.getRepresentedBy()))
+            .forEach(respondent -> representativesInfo.add(ImmutableMap.of(
```

Comment:
`Map.of` can be used instead of guava's `ImmutableMap.of`, but happy to leave it as it is (there are a lot of usages in code that could've been replaced with immutable map factory method introduced in Java 9)